$NetBSD: patch-aa,v 1.9 2005/11/18 13:03:53 rillig Exp $

The "gcc -o /dev/null" led to /dev/null be unlinked on Solaris.

--- configure.orig	Thu Oct 20 22:07:43 2005
+++ configure	Fri Nov 18 13:59:35 2005
@@ -21625,9 +21625,9 @@ fi;
 # Check whether --enable-jumbo-frames or --disable-jumbo-frames was given.
 if test "${enable_jumbo_frames+set}" = set; then
   enableval="$enable_jumbo_frames"
-  ac_enable_jumbo-frames="yes"
+  ac_enable_jumbo_frames="yes"
 else
-  ac_enable_jumbo-frames="no"
+  ac_enable_jumbo_frames="no"
 fi;
 
 # Check whether --enable-ipv6 or --disable-ipv6 was given.
@@ -22985,6 +22985,15 @@ case "${target}:${CC}" in
         echo "* This appears to be ${DEFINEOS} (release not checked)"
       ;;
 
+    *-*-netbsd*:* )
+	DEFINEOS="NETBSD"
+        echo "* This appears to be ${DEFINEOS}"
+      ;;
+    *-*-openbsd*:* )
+	DEFINEOS="OPENBSD"
+        echo "* This appears to be ${DEFINEOS}"
+      ;;
+
     * )
         echo "* ${target} is an unknown/unsupported OS"
         echo "*"
@@ -41690,8 +41699,6 @@ fi
 done
 
 
-case "${CC}" in
-  *gcc* )
         echo "$as_me:$LINENO: checking for mallinfo in malloc.h" >&5
 echo $ECHO_N "checking for mallinfo in malloc.h... $ECHO_C" >&6
     cat >conftest.$ac_ext <<_ACEOF
@@ -41709,9 +41716,7 @@ struct mallinfo memStats = mallinfo();
   return 0;
 }
 _ACEOF
-    gcc conftest.c -o /dev/null > /dev/null 2> /dev/null
-    rc=$?
-    if test ".${rc}" = ".0"; then
+    if eval "$ac_link"; then
         echo "$as_me:$LINENO: result: yes" >&5
 echo "${ECHO_T}yes" >&6
 
@@ -41723,7 +41728,6 @@ _ACEOF
         echo "$as_me:$LINENO: result: no" >&5
 echo "${ECHO_T}no" >&6
     fi
-esac
 
 # Expansion of NTOPCONFIGDEBUG_SETTINGS()
 if test ".${NTOPCONFIGDEBUG}" = ".yes"; then
